src/Controllers/PushController.js

```diff
@@ -49,6 +50,9 @@ export class PushController {
       onPushStatusSaved(pushStatus.objectId);
       return badgeUpdate();
     }).then(() => {
+      if (body.push_time) {
```

Comment:
That makes sense. I've updated the code to only allow skipping on the right configuration.